### PR TITLE
flow : use timestamp from the original packet

### PR DESCRIPTION
**Pull request by nplanel**

### CODE REVIEW

**Comment by safchain** [comment]
please see my comment in the RTT patch about replay

**Comment by nplanel** [comment]
run tests

**Comment by nplanel** [comment]
run tests

**Comment by nplanel** [comment]
run tests

**Comment by nplanel** [comment]
run tests

**Comment by nplanel** [comment]
run tests

**Comment by nplanel** [comment]
run skydive-functional-tests-backend-elasticsearch

**Comment by nplanel** [comment]
run skydive-functional-tests-backend-elasticsearch

**Comment by lebauce** [comment]
s/orignal/original

File: flow/flow.go [hunk 1]
```diff
@@ -680,8 +682,8 @@ func (f *Flow) newTransportLayer(packet *gopacket.Packet, tcpMetric bool) error
 
 // PacketsFromGoPacket split original packet into multiple packets in
 // case of encapsulation like GRE, VXLAN, etc.
-func PacketsFromGoPacket(packet *gopacket.Packet, outerLength int64, t int64, bpf *BPF) *Packets {
-	flowPackets := &Packets{Timestamp: t}
+func PacketsFromGoPacket(packet *gopacket.Packet, outerLength int64, bpf *BPF) *Packets {
+	flowPackets := &Packets{} //Timestamp: common.UnixMillis((*packet).Metadata().CaptureInfo.Timestamp)}
```
Comment (safchain) [inline flow/flow.go]: comment needs to be removed

Comment (nplanel) [reply flow/flow.go]: done

**Comment by safchain** [comment]
Still having the issue after the rebase. Please see the captures. I just used scale.sh with 3 pings

https://pasteboard.co/GVBx9Yj.png
https://pasteboard.co/GVBxtbn.png

**Comment by safchain** [comment]
could rebase it so that I could have another look of the aggregate issue

File: flow/table.go [hunk 2]
```diff
@@ -429,8 +419,6 @@ func (ft *Table) Run() {
 			if ok {
 				ft.reply <- ft.onQuery(query)
 			}
-		case now := <-nowTicker.C:
```
Comment (safchain) [inline flow/table.go]: Still testing the whole patch but here removing this ticker we trigger an issue as we have to wait an update or an expire timeout to get the ft.state checked. If have these parameters set to something high let say 5 min, we have to wait 5 mins before having to table stopped. So I propose to not remove the ticker.

Comment (nplanel) [reply flow/table.go]: agreed

Comment (nplanel) [reply flow/table.go]: I replaced it by a chan quit, as it make no sense to have a ticker just to exit a loop (evaluate more often)

Comment (safchain) [reply flow/table.go]: nice, thx

**Pull request merged**